mcs/class/corlib/System/TimeZoneInfo.cs

```diff
@@ -2604,7 +2671,7 @@ static internal TimeSpan GetUtcOffsetFromUtc (DateTime time, TimeZoneInfo zone,
 
 				// As we get the associated rule using the adjusted targetTime, we should use the adjusted year (targetTime.Year) too as after adding the baseOffset, 
 				// sometimes the year value can change if the input datetime was very close to the beginning or the end of the year. Examples of such cases:
-				//      �Libya Standard Time� when used with the date 2011-12-31T23:59:59.9999999Z
+				//      �Libya Standard Time� when used with the date 2011-12-31T23:59:59.9999999Z
```

Comment:
What's that?

Reply:
@TautvydasZilys Don't worry, I won't merge with this....Basically the file has some encoding problems and my vscode editor decided it would "help me out" and fix the incorrectly encoded characters and replace them with the unicode "can't display" character. Just easier to fix later when I'm ready to deliver than now.